ui/theme/theme.go

```diff
@@ -160,6 +170,26 @@ func (m *MyTheme) ListThemeFiles() map[string]string {
 	return result
 }
 
+func (m *MyTheme) createThemeResources() error {
+	err := configdir.MakePath(m.themeFileDir)
+	if err != nil {
+		return err
+	}
+
+	f, err := os.OpenFile(path.Join(m.themeFileDir, "default.toml"), os.O_WRONLY|os.O_CREATE, 0o644)
```

Comment:
This overwrites the default theme file every time the application is started, making `default.toml` essentially immutable. This may or may not be the desired behavior.

Reply:
Thanks for the PR! I'm on mobile now so I'll review it more closely when I'm back with PC access tomorrow, but just a high level comment that I don't think this is the desired behavior. I think we only want to create the default.toml file if it doesn't already exist. This allows the user to customize the default theme directly, and it can be "reset" back to its original state by deleting default.toml

Reply:
As you wish. My idea was to have a theme that is easy to copy (the default) being essentially immutable, so that it can serve as a template for custom themes. It's fine to drop it if it doesn't already exist, but I think it should be created automatically to prevent join issues with paths.